### DISCUSSION TITLE
Upgrade version doesn't work

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#13679] [Help] [answered]
Hi all,

I was trying to update the version from 2.20.10 to 2.26.1 but after the alert message that the updating was all ok actually the current version remains the old one.

I also tried to restart the budibase docker services.

Any suggestions?

Thank you

**Comment by mikesealey** [comment]
Have you tried clearing your browser cache?

**Comment by luigiPinnaAevo** [comment]
Hi @mikesealey,

yes, I deleted all data from the browser, same result.

**Comment by mikesealey** [comment, marked as answer]
In the command line, please try the following commands:
`docker-compose down`
`docker-compose up`

And then see if that remedies the issue.

**Comment by luigiPinnaAevo** [comment]
Ok done and problem solved. Thank you very much :)
